Documentation/bpf/progtypes.rst

```diff
@@ -338,49 +338,92 @@ corresponding network driver name of an interface can be determined as follows:
+|                        +-------------------------+-------------+
+|                        | otx2                    | >= 5.16     |
++------------------------+-------------------------+-------------+
+| Mediatek               | mtk                     | >= 6.0      |
++------------------------+-------------------------+-------------+
+| Mellanox               | mlx4                    | >= 4.8      |
+|                        +-------------------------+-------------+
+|                        | mlx5                    | >= 4.9      |
++------------------------+-------------------------+-------------+
+| Microchip              | lan966x                 | >= 6.1      |
++------------------------+-------------------------+-------------+
+| Microsoft              | hv_netvsc               | >= 5.6      |
+|                        +-------------------------+-------------+
+|                        | mana                    | >= 6.0      |
++------------------------+-------------------------+-------------+
+| Netronome              | nfp                     | >= 4.10     |
++------------------------+-------------------------+-------------+
+| Others                 | bonding                 | >= 5.15     |
+|                        +-------------------------+-------------+
+|                        | netdevsim               | >= 4.16     |
+|                        +-------------------------+-------------+
+|                        | tun/tap                 | >= 4.14     |
+|                        +-------------------------+-------------+
+|                        | virtio_net              | >= 4.10     |
+|                        +-------------------------+-------------+
+|                        | xen-netfront            | >= 5.9      |
+|                        +-------------------------+-------------+
+|                        | Veth                    | >= 4.19     |
++------------------------+-------------------------+-------------+
+| Qlogic                 | qede                    | >= 4.10     |
++------------------------+-------------------------+-------------+
+| SFC                    | Efx                     | >= 5.5      |
++------------------------+-------------------------+-------------+
+| Socionext              | netsec                  | >= 5.3      |
++------------------------+-------------------------+-------------+
+| Solarflare             | sfc                     | >= 5.5      |
++------------------------+-------------------------+-------------+
+| STMicro                | stmmac                  | >= 5.3      |
++------------------------+-------------------------+-------------+
+| Texas Instruments      | cpsw                    | >= 5.3      |
++------------------------+-------------------------+-------------+
+| VMware                 | vmxnet3                 | >= 6.6      |
++------------------------+-------------------------+-------------+
+| Xen                    | xennet                  | >= 5.9      |
```

Comment:
Isn't that one the same as `xen-netfront` in the “Others” section?

Reply:
ah yes, both are same.